ucac/package-lock.json

```diff
@@ -4,56 +4,39 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
-    "acorn": {
-      "version": "5.1.2",
-      "resolved": "https://registry.npmjs.org/acorn/-/acorn-5.1.2.tgz",
-      "integrity": "sha512-o96FZLJBPY1lvTuJylGA9Bk3t/GKPPJG8H0ydQQl01crzwJgspa4AEIq/pVTXigmK0PHVQhiAtn8WMBLL9D2WA==",
+    "@sindresorhus/is": {
```

Comment:
Since this PR doesn't explicitly update any packages, let's exclude this lockfile from the PR. We can handle dependency/lockfile updates in a follow-up PR.